ocp_resources/resource.py

```diff
@@ -1368,7 +1368,7 @@ def _get() -> Generator["NamespacedResource|ResourceInstance", None, None]:
                         yield cls(
                             client=dyn_client,
                             name=resource_field.metadata.name,
-                            namespace=resource_field.metadata.namespace,
+                            namespace=str(resource_field.metadata.namespace),
```

Comment:
This issue seems like an user error situation. namespace here is namespace name. This is true for all wrapper objects, if I recall correctly. Just like an oc command does not work with kinds as params, wrapper expects namespace strings when you are providing a namespace.